okhttp/src/main/java/okhttp3/Response.kt

```diff
@@ -302,6 +302,21 @@ class Response internal constructor(
     checkNotNull(body) { "response is not eligible for a body and must not be closed" }.close()
   }
 
+  /**
+   * Returns the sequence of Responses that led to this final Response.
```

Comment:
Is this backwards? I think `this` should be last for most uses? Also unsure of the name. response.responses() feels non-obvious. I wanna reach for something like `history()` but that doesn’t feel right either. `progression()` ? That’s too scary-sounding.